src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -0,0 +1,261 @@
+          programName: this.program,
+          campusLocation: this.campusLocation,
+          studentName: this.studentName,
+          studentNumber: this.studentNumber,
+          languageofInstruction: this.language,
+          startDate: this.startYear,
+          endDate: this.endYear,
+        };
+        const clientId = this.clientId ? this.clientId : this.localId;
+        // Update the modelValue dictionary
+        const updatedModelValue = {
+          ...this.modelValue,
+          [clientId]: newTranscript,
+        };
+
+        // Emit the updated modelValue
+        this.$emit("update:model-value", updatedModelValue);
+
+        // Set success alert message
+        const message = this.modelValue[clientId] ? "You have successfully edited your Education." : "You have successfully added your Education.";
+        this.alertStore.setSuccessAlert(message);
+
+        // Change mode to education list
+        this.mode = "list";
+      } else {
+        this.alertStore.setFailureAlert("Please fill out all required fields");
+      }
+    },
+    handleCancel() {
+      // Change mode to education list
+      this.mode = "list";
+    },
+    handleAddEducation() {
+      // Reset the form fields
+      this.resetFormData();
+      this.clientId = "";
+      this.mode = "add";
+    },
+    handleEdit(educationData: EducationData) {
+      console.log("Edit education with id: ", educationData.educationId);
+      console.log("Education data: ", educationData.education);
+      // Set the form fields to the education data
+      this.id = educationData.education.id ?? "";
+      this.clientId = educationData.educationId.toString();
+      this.previousSchool = educationData.education.educationalInstitutionName ?? "";
+      this.school = educationData.education.educationalInstitutionName ?? "";
+      this.program = educationData.education.programName ?? "";
+      this.campusLocation = educationData.education.campusLocation ?? "";
+      this.studentName = educationData.education.studentName ?? "";
+      this.studentNumber = educationData.education.studentNumber ?? "";
+      this.language = educationData.education.languageofInstruction ?? "";
+      this.startYear = educationData.education.startDate ?? "";
+      this.endYear = educationData.education.endDate ?? "";
+      // Change mode to add
+      this.mode = "add";
+    },
+    handleDelete(educationId: string | number) {
+      console.log("Delete education with id: ", educationId);
+      //Remove the education from the modelValue
+
+      if (educationId in this.modelValue) {
+        // Create a copy of modelValue
+        const updatedModelValue = { ...this.modelValue };
+        // Delete the education entry from the copied object
+        delete updatedModelValue[educationId];
+        // Emit the updated modelValue
+        this.$emit("update:model-value", updatedModelValue);
+      }
+
+      this.alertStore.setWarningAlert("You have Deleted your Education.");
```

Comment:
I feel like we don't need to give a warningAlert to the user. 

Technically they haven't deleted their education. They need to save draft first.

Reply:
It was already there if It's not in requirements we can remove it

Reply:
@SoLetsDev let's leave it for now to match requirements/design

![Screenshot 2024-03-07 at 10 40 03 AM](https://github.com/bcgov/ECC-ECER/assets/10508230/5753bbb0-70ef-4ba2-9cfd-e5dd4c632641)

Reply:
sounds good to be